docs/design/fabric-v2+/fpc-management.md

```diff
@@ -76,28 +76,33 @@ Users must aware that each FPC chaincode runs in a single enclave at a designate
 
-The initial version of FPC restricts the set of allowed endorsement policies and currently
-supports a designated endorser policy only. That is, only a single endorsing peer is
-responsible to execute transactions of an FPC chaincode for the consortium.  As an FPC
-chaincode is protected by the use of Trusted Execution Technology, from a security
-perspective, the integrity guarantees gained from the endorsement model are similar to the guarantee provided in a typical Fabric system with multiple Peer endorsements.
-However, the designated endorser policy does not provide resilience in terms of availability and limits a particular chaincode's scalability.
-In future releases of FPC, we will address this limitation and relax this restriction by
-providing support for rich endorsement policies.
-
-To achieve the designated-peer endorsement policy, the consortium initially selects an organization (e.g., SampleOrg)
-that will host the designated endorsing peer.  The corresponding endorsement policy
-needs to be set to `OR('SampleOrg.peer')` and approved by the consortium using normal chaincode lifecycle agreement.
-Other expressions like `AND` or `OR` with multiple organizations are currently not allowed and
-may lead to unexpected behavior.
-
-Note that in order to use `SampleORG.peer`, `NodeOUs.Enable` must be set to `true`, otherwise
-`OR('SampleOrg.member')` must be used.
-
-For example:
-
-    peer lifecycle chaincode approveformyorg --channelID mychannel --name myfpc --signature-policy "OR('SampleOrg.peer')" ...
-
-An admin of SampleOrg is then responsible to invoke `lifecycle chaincode initEnclave` at
-a single peer and thereby determine the designated endorser for the `myfpc` chaincode.  Clients
-of the consortium have to make sure that they invoke transactions only at the designed endorser.
+In FPC, signatures by enclaves are the base of endorsements.
+For the initial version of FPC, FPC Lite, an endorsement of a single
+endorsing peer is sufficient.
+More specifically, the endorsement must stem from an enclave which is properly registered with the enclave registiry (ercc) as part of a successful `initEnclave` call (see [above](#create-chaincode-enclave).
+The corresponding *enclave endorsement policy* is called `designated chaincode` and is *implicitly defined*.
+As an FPC chaincode is protected by the use of Trusted Execution Technology,
+from a security perspective,
+the integrity guarantees gained from a single enclave are similar to the guarantee provided in a typical Fabric system with multiple peer endorsements.
+
+With the [externalized enclave endorsement validation](https://docs.google.com/document/d/1RSrOfI9nh3d_DxT5CydvCg9lVNsZ9a30XcgC07in1BY/)
+of the initial version of FPC,
+there is though also separate endorsement required,
+namely an endorsement that the enclave endorsement was properly validated.
+The policy is called the validation endorsement policy* and should match the standard organizational trust model in Fabric,
```

Comment:
```suggestion
The policy is called the **validation endorsement policy** and should match the standard organizational trust model in Fabric,
```

Reply:
ah, a '*' got lost. but will leave it as `*` rather than `**` to be consistent with other similar uses ...